pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+var getpvlist = `
+kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
+
+func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace string) []string {
+
+	getpv := fmt.Sprintf(getpvlist)
```

Comment:
For readability, I'd suggest just putting the command directly here, so the dev doesn't need to look above to find what is executed.
```suggestion
	getpv := `kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
```